06-application-observability.md

```diff
@@ -10,13 +10,13 @@ In this step, we will use a more complex serverless application. To deploy it, s
 cd ../wild-rydes-workshop/
 git fetch
 git checkout monitoring
-cd -
```

Comment:
I think we still want this? Without it, I'll think we'll be in the wrong directory when we do the next command...

Reply:
I meant to add `cd ..` - good catch!